tools/gen.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
License Header?

Reply:
Added. Although license headers are very varied across all files in Google GitHub projects.